src/components/PortForwarding.vue

```diff
@@ -70,8 +88,21 @@ export default {
     };
   },
   computed: {
+    notRunning() {
+      return this.k8sState !== K8s.State.STARTED;
+    },
     rows() {
-      return this.services.map(service => ({
+      let services = this.services;
+
+      if (!this.includeKubernetesServices) {
+        services = services.filter((service) => {
+          return (service.namespace !== 'kube-system' &&
+            !(service.namespace === 'default' && service.name === 'kubernetes')
+          );
+        });
```

Comment:
This may be clearer as two filters?
```suggestion
        services = services
          .filter((service) => service.namespace !== 'kube-system')
          .filter((service) => !(service.namespace === 'default' && service.name === 'kubernetes'));
```
(I'm not actually sure…)

Reply:
sure